apps/frontend/src/libs/components/popover/popover.tsx

```diff
@@ -2,36 +2,43 @@ import {
 	useCallback,
 	useHandleClickOutside,
 	useRef,
-	useState,
 } from "~/libs/hooks/hooks.js";
 
 import styles from "./styles.module.css";
 
 type Properties = {
 	children: React.ReactNode;
 	content: React.ReactNode;
+	isPopoverOpened: boolean;
+	onPopoverClose: () => void;
+	onPopoverOpen: () => void;
 };
 
-const Popover = ({ children, content }: Properties): JSX.Element => {
-	const [isOpened, setIsOpened] = useState<boolean>(false);
+const Popover = ({
+	children,
+	content,
```

Comment:
I don't think that now, when we have `usePopover` hook that controls visibility of popover we need `content` prop. Now we don't want to wrap a component that should response for click and open popover with `Popover` component. Instead of this, `Popover` will take only children, representing its content, and it will be placed near the component it's belong to. 

Hm, however I'm not sure how then we will position it where it suppose to be. @liza-veis Can you suggest something, or say if my suggestion is not good

Reply:
We need to have content to position it correctly and also we have additional wrappers for the popover and it's content